pkg/apis/core/v1beta1/defaults/defaults.go

```diff
@@ -87,10 +90,20 @@ func SetDefaultKubeFedConfig(fedConfig *v1beta1.KubeFedConfig) {
 		spec.SyncController = &v1beta1.SyncControllerConfig{}
 	}
 
+	syncController := spec.SyncController
+	setInt64(&syncController.MaxConcurrentReconciles, DefaultSyncControllerMaxConcurrentReconciles)
+
 	if spec.SyncController.AdoptResources == nil {
 		spec.SyncController.AdoptResources = new(v1beta1.ResourceAdoption)
 		*spec.SyncController.AdoptResources = v1beta1.AdoptResourcesEnabled
 	}
+
+	if spec.StatusController == nil {
+		spec.StatusController = &v1beta1.StatusControllerConfig{}
+	}
+
+	statusController := spec.StatusController
+	setInt64(&statusController.MaxConcurrentReconciles, DefaultStatusControllerMaxConcurrentReconciles)
```

Comment:
nit: no need to introduce variable here
```suggestion
	setInt64(&spec.StatusController.MaxConcurrentReconciles, DefaultStatusControllerMaxConcurrentReconciles)
```